Source/Paths/SvgArcSegment.cs

```diff
@@ -135,7 +136,7 @@ public override string ToString()
         {
             var arcFlag = Size == SvgArcSize.Large ? "1" : "0";
             var sweepFlag = Sweep == SvgArcSweep.Positive ? "1" : "0";
-            return "A" + RadiusX.ToString() + " " + RadiusY.ToString() + " " + Angle.ToString() + " " + arcFlag + " " + sweepFlag + " " + End.ToSvgString();
+            return "A" + RadiusX.ToString(CultureInfo.InvariantCulture) + " " + RadiusY.ToString(CultureInfo.InvariantCulture) + " " + Angle.ToString(CultureInfo.InvariantCulture) + " " + arcFlag + " " + sweepFlag + " " + End.ToSvgString();

```

Comment:
Thanks! While you are it - could you check for more occurencies and fix them, too? I did a quick check, and found at least a couple more in `SvgOrient.ToString` and in `SvgOrientConverter.ConvertFrom`.

Reply:
Fixed SvgOrient.ToString, not sure if SvgOrientConverter.ConvertFrom needs that.

Reply:
I have checked other ToString methods and could not find any.

Reply:
> not sure if SvgOrientConverter.ConvertFrom needs that.

I meant this line:
```C#
      if (!float.TryParse(value.ToString(), out fTmp))
```
As far as I know, this uses the current locale if not given one.

Reply:
But maybe I'm wrong here... for some reason the culture info is given as an argument in `ConvertFrom` here, but not used.

Reply:
Maybe we should leave this for now, as I am not convinced this needs invariant culture.

Reply:
Agreed. Thanks for the help!